repository/src/test/java/org/jbpm/process/workitem/repository/service/RepoServiceIntegrationTest.java

```diff
@@ -122,4 +126,25 @@ public void testInstalledUninstalledModules() throws Exception {
         assertEquals(0,
                      installeddRepoModules.size());
     }
+
```

Comment:
I expect remove operation to be tested here.

Reply:
line 143, it is a test for remove operation.